packages/sync/src/Actions.php

```diff
@@ -132,34 +199,72 @@ static function sync_allowed() {
-	static function sync_via_cron_allowed() {
+	/**
+	 * Determines if syncing during a cron job is allowed.
+	 *
+	 * @access public
+	 * @static
+	 *
+	 * @return bool|int
+	 */
+	public static function sync_via_cron_allowed() {
 		return ( Settings::get_setting( 'sync_via_cron' ) );
 	}
 
-	static function prevent_publicize_blacklisted_posts( $should_publicize, $post ) {
-		if ( in_array( $post->post_type, Settings::get_setting( 'post_types_blacklist' ) ) ) {
+	/**
+	 * Decides if the given post should be Publicized based on its type.
+	 *
+	 * @access public
+	 * @static
+	 *
+	 * @param bool     $should_publicize  Publicize status prior to this filter running.
+	 * @param \WP_Post $post              The post to test for Publicizability.
+	 * @return bool
+	 */
+	public static function prevent_publicize_blacklisted_posts( $should_publicize, $post ) {
+		if ( in_array( $post->post_type, Settings::get_setting( 'post_types_blacklist' ), true ) ) {
 			return false;
 		}
 
 		return $should_publicize;
 	}
 
-	static function set_is_importing_true() {
+	/**
+	 * Set an importing flag to `true` in sync settings.
+	 *
+	 * @access public
+	 * @static
+	 */
+	public static function set_is_importing_true() {
 		Settings::set_importing( true );
 	}
 
-	static function send_data( $data, $codec_name, $sent_timestamp, $queue_id, $checkout_duration, $preprocess_duration ) {
+	/**
+	 * Sends data to WordPress.com via an XMLRPC request.
+	 *
+	 * @access public
+	 * @static
+	 *
+	 * @param object $data                   Data relating to a sync action.
+	 * @param string $codec_name             The name of the codec that encodes the data.
+	 * @param float  $sent_timestamp         Current server time so we can compensate for clock differences.
+	 * @param string $queue_id               The queue the action belongs to, sync or full_sync.
+	 * @param float  $checkout_duration      Time spent retrieving queue items from the DB.
+	 * @param float  $preprocess_duration    Time spent converting queue items into data to send.
+	 * @return Jetpack_Error|mixed|WP_Error  The result of the sending request.
+	 */
+	public static function send_data( $data, $codec_name, $sent_timestamp, $queue_id, $checkout_duration, $preprocess_duration ) {
 		\Jetpack::load_xml_rpc_client();
 
 		$query_args = array(
-			'sync'      => '1',             // add an extra parameter to the URL so we can tell it's a sync action
-			'codec'     => $codec_name,     // send the name of the codec used to encode the data
-			'timestamp' => $sent_timestamp, // send current server time so we can compensate for clock differences
-			'queue'     => $queue_id,       // sync or full_sync
-			'home'      => Functions::home_url(),  // Send home url option to check for Identity Crisis server-side
-			'siteurl'   => Functions::site_url(),  // Send siteurl option to check for Identity Crisis server-side
-			'cd'        => sprintf( '%.4f', $checkout_duration ),   // Time spent retrieving queue items from the DB
```

Comment:
I'd say these comments were useful.

Reply:
I've removed only the redundant comments that are already present in the doc block